src/Filters/atoms/DateTimeBlock.tsx

```diff
@@ -178,6 +192,14 @@ const DateTimeBlock : React.FC<IProps> = ({
     }
   }, []);
 
+  const onKeyDownChange = useCallback((event) => {
+    if(event.key==='.' || event.key==='e' || event.key==='-'){
```

Comment:
I can see why you're stopping `.` from firing here, but I couldn't replicate `e` or `-` causing any problems in these fields at all. Are these needed for a reason I am not realising?

Reply:
Updated code.